### PR TITLE
Fix #313 just check if there's middle components, if not then we show the popover

### LINKED ISSUE #313: Create Integration: Popover Should Remain Open First Time
This currently requires that the user hover over the plus icon, but should remain open by default for the first time.

screenshot 2017-03-24 11 05 58

Design:

save or add step

Ref: https://redhat.invisionapp.com/share/VQ8UZ7PCE#/224783021_IPaaS_CreateIntegration_MultiStep-pdf_2

### PR DESCRIPTION
**Pull request by gashcrumb**
This also reduces the height of steps so there's less vertical used by a flow, also tweaks some highlight behavior.

### CODE REVIEW

**Comment by kahboom** [comment]
@gashcrumb - Are your tests passing locally? They seem to have failed on CircleCI. :(

**Comment by gashcrumb** [comment]
@kahboom lint once again :-)

File: src/app/integrations/edit-page/flow-view/flow-view-step.component.scss [hunk 1]
```diff
@@ -91,6 +91,7 @@
       //----- Sidebar: Progress Line: Complete State ----------------->>
 
       &.complete {
+        border: 3px solid $pf-blue-400;
```
Comment (kahboom) [inline src/app/integrations/edit-page/flow-view/flow-view-step.component.scss]: Woo!

File: src/app/integrations/edit-page/flow-view/flow-view.component.scss [hunk 2]
```diff
@@ -135,10 +135,8 @@ $flow-view-container-min-height: 0;
 
   .steps {
     position: relative;
-    //height: 150px;
-    height: 100%;
     flex-wrap: nowrap;
-    min-height: 250px;
+    min-height: 150px;
```
Comment (kahboom) [inline src/app/integrations/edit-page/flow-view/flow-view.component.scss]: Sorry about that, just wanted it to be spaced out more on a larger screen.. maybe we can address that later on.

**Pull request merged**